docs/product-assembly_designing-and-building-cases/src/main/asciidoc/case-management-close-case-using-API-proc.adoc

```diff
@@ -0,0 +1,32 @@
+[id='case-management-close-case-using-API-proc-{context}']
+= Closing a case using REST APIs
```

Comment:
Maybe "...using the {KIE_SERVER} REST API" or "...REST API requests"? Technically it's a single API that they're using, with many supported requests.